### ISSUE TITLE
rsyslog/syslog_appliance_alpine - volume grows without stop - how to solve

### ISSUE THREAD

**Issue by francesco-russo-nanabianca-it** [rsyslog/rsyslog-docker#46]
Hi I'm new here, I'm just using that image for catch all logs from my connected device.

Everythink works as expected, except that the volume used by this container /var/lib/docker/volumes/xxx_xxx_xxx_xxx_xxx_xxx_xxx_xxx_xxx_xxx/ is continously growing until my disk is full and block my server.

The only solution I found for now is to stop the container, delete this volumes and restart the container.

Any best/more elegant/practice???

Thanks to everyone will answer me

**Comment by igoras1993** [comment]
Can You please post Your config and the exact command / docker-compose You have used to run the container?

**Comment by francesco-russo-nanabianca-it** [comment]
Yes of course, I just run the container with the following command:
`run -it --name logging_container -p 514:514 -p 514:514/udp -v /var/log/rsyslogd:/logs/hosts rsyslog/syslog_appliance_alpine`
